spec/OBD/OBD.vspec

```diff
@@ -33,9 +34,10 @@
 #
 
 PidsA:
-  datatype: uint32
-  type: sensor
-  description: PID 00 - Bit array of the supported pids 01 to 20
+  datatype: string[]
+  type: attribute
+  allowed: ["01",	"02", "03",	"04",	"05",	"06",	"07",	"08",	"09",	"0A",	"0B",	"0C",	"0D",	"0E",	"0F",	"10",	"11",	"12",	"13",	"14",	"15",	"16",	"17",	"18",	"19",	"1A",	"1B",	"1C",	"1D",	"1E",	"1F",	"20"]
+  description: PID 00 - Bit array of the supported pids 01 to 20 in Hexadecimal.
```

Comment:
Previously it was a bit array, i.e. a 32 bit uint where each bit represented a pid. Now it is a real array instead so shall better be "PID 00 - Array of ..."

By the way - if we do this change, should we then not change PidsB and PidsC as well?

Reply:
Done. Pids B and C were updated to match the definition of A.